### PR TITLE
IMP Create new computed stored field 'display_name' that use zip, city, ...

### PR DESCRIPTION
**Pull request by jjscarafia**
...state and country in order to allow search using '%' from the m2o widget

### CODE REVIEW

File: base_location/models/better_zip.py [hunk 1]
```diff
@@ -36,32 +38,22 @@ class BetterZip(models.Model):
...
-        return res
+    @api.one
+    @api.depends(
+        'name',
+        'city',
+        'state_id',
+        'country_id',
+        )
+    def _get_display_name(self):
+        self.display_name = ('(%s) %s, %s, %s') % (
```
Comment (pedrobaeza) [inline base_location/models/better_zip.py]: Please don't change the current behaviour of name_get for this field. Use:

```
if self.name:
    name = [bzip.name, bzip.city]
else:
    name = [bzip.city]
if bzip.state_id:
    name.append(bzip.state_id.name)
if bzip.country_id:
    name.append(bzip.country_id.name)
self.display_name = ", ".join(name)
```

**Comment by jjscarafia** [comment]
Now is done. Thanks @pedrobaeza !

**Comment by jjscarafia** [comment]
Any news on this PR? Should I make anything? If you find is not suitable I can make this mods on our own branch, just let me know. 
Thanks!

**Comment by pedrobaeza** [comment]
For me, :+1: 

We need another review for merging.

**Comment by jjscarafia** [comment]
ahother review here please :)

**Comment by foutoucour** [comment]
:+1:

**Comment by jjscarafia** [comment]
Thanks!

**Pull request merged**